client/state/jetpack-connect/reducer.js

```diff
@@ -36,29 +36,39 @@ import {
 	DESERIALIZE
 } from 'state/action-types';
 
+import { isValidStateWithSchema } from 'state/utils';
+import { jetpackConnectSessionsSchema } from './schema';
+
 const defaultAuthorizeState = {
 	queryObject: {},
 	isAuthorizing: false,
 	authorizeSuccess: false,
 	authorizeError: false
 };
 
-function buildNoProtocolUrlObj( url ) {
+function buildNoProtocolUrlObj( url, isInstall ) {
```

Comment:
Hmm, well, since we are sharing data storage, technically even if the info is added by the SSO flow, it could be used in the jetpack connect basic one, so it would need to have the 'isInstall' param. It's not a huge difference, because in that cases that param is always going to be `false` and it won't matter if its defined or not, but for data normalization, I would rather add it anyway so we don't have extra logic to avoid storing a boolean